app/views/admin/editions/_show_metadata.html.erb

```diff
@@ -51,6 +51,12 @@
     selected_taxon_paths: @edition_taxons.map(&:full_path),
     no_topics_message: 'No topics - please add a topic before publishing'
   } %>
+<% else %>
+  <%= render partial: '/admin/shared/tagging/legacy_associations_box', locals: {
+      path_to_edit_associations: edit_admin_edition_legacy_associations_path(@edition.id),
+      edition: @edition,
+      no_policy_areas_message: 'No policy areas - please add a policy area before publishing'
```

Comment:
The `no_policy_areas_message` is only displayed when the edition can be tagged to policy areas. This means that for doc types which can't (I found case studies as an example), no message is shown:

<img width="786" alt="screen shot 2018-05-24 at 11 33 49" src="https://user-images.githubusercontent.com/29889908/40480592-e6678308-5f46-11e8-8047-6a6c226c43cf.png">

Not sure if this is intended or if we want to add a message like "No legacy associations", similar to what we do for translations (as shown).

Reply:
I've created a new card to deal with this one. I don't want to hold off getting this one merged as it's holding us all back.